src/features/common/libs/utils/utils.test.ts

```diff
@@ -0,0 +1,47 @@
+import { createSearchParams } from "react-router-dom";
+import { searchParamsToObject, withoutFalsyValues } from "./utils";
+
+describe("utils", () => {
+  beforeAll(() => jest.useFakeTimers());
+  afterAll(() => jest.useRealTimers());
+
+  it("searchParamsToObject() should work as expected", async () => {
```

Comment:
Make tests more desciptive: e.g: "searchParamsToObject should convert search params string into object"